rpc/src/v1/types/uint.rs

```diff
@@ -48,6 +49,31 @@ macro_rules! impl_uint {
 			}
 		}
 
+		impl $name {
+			/// Human readable formatting
+			pub fn format_ether(&self) -> String {
```

Comment:
The problem is, the wrapper type does not expose its inner value, or any of the arithmetic operations, i'm not sure if it would be cleaner to make the wrapped type public? opinions? @rphmeier @gavofyork